test-configs.yaml

```diff
@@ -1202,6 +1212,11 @@ device_types:
     class: arm-dtb
     boot_method: uboot
 
+  sun8i-h3-bananapi-m2-plus:
+    mach: allwinner
+    class: arm-dtb
+    boot_method: uboot
```

Comment:
Tested OK on staging: https://staging.kernelci.org/test/plan/id/5e982048adbfd8b6dcb2b416/
Device in upstream LAVA.